src/modules/blocks/event-datetime/controls/container.js

```diff
@@ -11,33 +11,36 @@ import {
 	actions as dateTimeActions,
 	selectors as dateTimeSelectors,
 } from '@moderntribe/events/data/blocks/datetime';
-import { withStore, withSaveData, withBlockCloser } from '@moderntribe/common/hoc';
+import { withStore } from '@moderntribe/common/hoc';
 import EventDateTimeControls from './template';
 
 /**
  * Module Code
  */
 
-const onTimeZoneVisibilityChange = ( dispatch ) => ( checked ) => (
-	dispatch( dateTimeActions.setTimeZoneVisibility( checked ) )
-);
-
 const mapStateToProps = ( state ) => ( {
 	isEditable: dateTimeSelectors.isEditable( state ),
 	separatorDate: dateTimeSelectors.getDateSeparator( state ),
 	separatorTime: dateTimeSelectors.getTimeSeparator( state ),
-	showTimeZone: dateTimeSelectors.getTimeZoneVisibility( state ),
 	timeZone: dateTimeSelectors.getTimeZone( state ),
 } );
 
-const mapDispatchToProps = ( dispatch ) => ( {
-	...bindActionCreators( dateTimeActions, dispatch ),
-	onTimeZoneVisibilityChange: onTimeZoneVisibilityChange( dispatch ),
+const mapDispatchToProps = ( dispatch, ownProps ) => ( {
```

Comment:
I'm a fan of destructuring whenever possible. Just coding style here, doesn't really matter especially if inconsistent with the rest of the project.

```suggestion
const mapDispatchToProps = ( dispatch, { setDateTimeAttributes } ) => ( {
```

and then down below `setDateTimeAttributes( { separatorDate: value } )`